packages/main/src/SliderBase.ts

```diff
@@ -108,6 +109,17 @@ abstract class SliderBase extends UI5Element {
 	@property({ type: Boolean })
 	showTooltip = false;
 
+	/**
+	 *
+	 * Indicates whether input fields should be used as tooltips for the handles. Note: Setting this option to true will only work if showTooltip is set to true.
```

Comment:
Here is a "Note:" that is not bolded. Should every note be bolded and on separate row or this could stays like taht? @LilyanaOviPe